components/rule-mgt/org.wso2.carbon.identity.rule.metadata/src/main/java/org/wso2/carbon/identity/rule/metadata/config/FlowConfig.java

```diff
@@ -90,4 +104,81 @@ private static void validateFlow(String flowName) throws RuleMetadataConfigExcep
             throw new RuleMetadataConfigException("Invalid flow: " + flowName);
         }
     }
+
+    /**
+     * Resolve the field definition from the field definition config.
+     *
+     * @param fieldDefinitionConfig FieldDefinitionConfig
+     * @param entry                 Entry
+     * @return FieldDefinition
+     * @throws RuleMetadataConfigException If an error occurs while resolving the field definition
+     */
```

Comment:
shall we not have these doc comments for private methods unless it's really necessary.
Because majority of the time we will be updating private methods and eventually these becomes stale or another hop to update.

Reply:
Fixed